backend/src/auth/test/TestMockUserStore.ts

```diff
@@ -0,0 +1,53 @@
+import { UserModel } from "../data_model/UserModel";
```

Comment:
Thanks for adding tests!

Reply:
One thing we'll need to do is find a good testing framework